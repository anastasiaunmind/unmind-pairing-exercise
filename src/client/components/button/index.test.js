import React from "react";
import { render } from "@testing-library/react";
import Button from ".";

test("renders button", () => {
    const { container } = render(<Button text="demo" onClick={() => {}} />);

    expect(container).toMatchSnapshot();
});
