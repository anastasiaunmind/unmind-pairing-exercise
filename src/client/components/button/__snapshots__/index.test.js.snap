// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button 1`] = `
<div>
  <button
    class="sc-bdnxRM gZiVar"
  >
    demo
  </button>
</div>
`;
